pkg/apis/core/v1/conversion.go

```diff
@@ -382,6 +385,10 @@ func Convert_core_Pod_To_v1_Pod(in *core.Pod, out *v1.Pod, s conversion.Scope) e
 	// remove this once the oldest supported kubelet no longer honors the annotations over the field.
 	out.Annotations = dropInitContainerAnnotations(out.Annotations)
 
+	if err := HandleSeccompVersionSkew(in, out); err != nil {
+		return errors.Wrap(err, "handling seccomp version skew")
+	}
```

Comment:
Conversion is not the right place for this... an error here means the pod won't even reach validation code in the API server.

podStrategy#PrepareForCreate and podStrategy#PrepareForUpdate are where I would expect us to make these changes

Reply:
Thank you for pointing me to the right direction, I wasn't quite sure where in the code that was to take place. PATL

Reply:
I still don't see anything in podStrategy#PrepareForCreate or podStrategy#PrepareForUpdate that does the annotation<->field population